source/2020-02-21-the-ember-times-issue-136.md

```diff
@@ -44,13 +45,15 @@ A **new** version of Ember is here! [Ember 3.16](https://blog.emberjs.com/2020/0
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Tomsweeper: Minesweeper meets Ember Octane 💣](https://tomsweeper.scud.co/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Minesweeper is a classic logic game, probably most known through Microsoft Windows. You use the number clues to clear the mines and race against time to [become the world champion](https://www.youtube.com/watch?v=GrZCWx0fnfc).
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@scudco](https://github.com/scudco) issues a new challenge: Find all Tomsters in the game [Tomsweeper](https://tomsweeper.scud.co/)! It's a great example of how you can [build something in Ember Octane](https://github.com/scudco/tomsweeper) with ease and fun.
```

Comment:
I couldn't find their name either, but if you happen to know it you might include that.

Reply:
Yeah, I tried finding their name, but because they didn't specify it on GitHub or Twitter, I figured they may want to remain anonymous.